ref/general/microshed.adoc

```diff
@@ -3,12 +3,12 @@
 :page-description: MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 :page-categories: MicroShed testing
 :seo-title: Testing in a container with MicroShed testing
-:seo-description: MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
+:seo-description:  MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 = Testing in a container
 
-MicroShed testing helps you write integration tests by using Testcontainers for Java microservice applications. With MicroShed testing, you can test your Open Liberty application from outside the container so that you test the exact same image that runs in production.
+MicroShed testing helps you to write integration tests using Testcontainers for Java microservice applications. With MicroShed testing you test your application in the same container that you use in production, thus reducing the risk of test failure.
```

Comment:
Change "write integration tests using Testcontainers" to "write integration tests by using Testcontainers".